include/oneapi/dpl/type_traits

```diff
@@ -218,10 +215,9 @@ using ::std::is_volatile_v;
 using ::std::negation;
 using ::std::rank_v;
 using ::std::void_t;
-#    if __cplusplus == 201703L
-using ::std::is_literal_type_v;
-#    endif
-#endif // __cplusplus > 201402L
+#if __cplusplus < 202002L
+using ::std::is_literal_type_v; // Deprecated in C++17, removed in C++20
+#endif                          // __cplusplus < 202002L
```

Comment:
[[unimportant/ignorable]] I would put all 4 deprecated names next to each other, instead of spreading over the list.

Reply:
Fixed.